javascript/src/main/components/ChannelMessages/ChannelLinks.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import {useAuth0} from "@auth0/auth0-react";
+import useSWR from "swr";
+import {fetchWithToken} from "main/utils/fetch";
+import ToolkitProvider, { Search } from 'react-bootstrap-table2-toolkit';
+import TimeFormatter from "./time"
+const { SearchBar } = Search;
+
+const GetMessageContents = (text, slackUsers) => {
+    return text.replace(/<@([A-Z0-9]{11})>/g, (_,userId) => {
+        if(slackUsers != null) {
+            for(let i = 0; i < slackUsers.length; i++) {
+                if(slackUsers[i].id === userId) {
+                    return "@" + slackUsers[i].real_name;
+                }
+            }
+        }
+        return "@" + userId;
+    })
+}
+
+const filterLinks = (messages) => {
+    // get bracketed text (links)
+    var link_messages = messages.filter(message => (message.text.includes("<") && message.text.includes(">")));
+    var bracketRegEx = /<(.*?)>/g;
+    var links = [];
+    var found;
+    var new_message;
```

Comment:
Can we use `let` or `const` instead of `var`? It's better practice and it solves issues with scoping.